core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/MethodConfigResolver.scala

```diff
@@ -53,8 +51,14 @@ object MethodConfigResolver {
   case class MethodInput(workflowInput: WorkflowInput, expression: String)
 
   def gatherInputs(methodConfig: MethodConfiguration, wdl: String): Try[Seq[MethodInput]] = parseWDL(wdl) map { workflow =>
+    def isAttributeEmpty(fqn: FullyQualifiedName): Boolean = {
+      methodConfig.inputs.get(fqn) match {
+        case Some(AttributeString(value)) => value.isEmpty
+        case _ => throw new AssertionError(s"MethodConfiguration ${methodConfig.namespace}/${methodConfig.name} input ${fqn} value is unavailable")
```

Comment:
You might want to throw `RawlsException` here rather than `AssertionError`. AssertionError is an `Error` not an `Exception`, so it might bubble up where you don't expect it to. `RawlsException` should be handled by the application.